package.json

```diff
@@ -112,5 +112,8 @@
     "*.json": "prettier --write",
     "*.yml": "prettier --write",
     "*.md": "prettier --write"
+  },
+  "overrides": {
+    "minimatch": "10.0.1"
```

Comment:
this should go to deployer (where dependency on toolkit is). otherwise won't be visible outside of our repo.